src/Symfony/Component/HttpFoundation/ResponseHeaderBag.php

```diff
@@ -229,7 +229,7 @@ public function removeCookie($name, $path = '/', $domain = null)
      *
      * @param string $format
      *
-     * @return array
+     * @return Cookie[] An array of cookies
```

Comment:
this type is wrong. It depends on the format. It is true for the flat format, but not for the array format.